### ISSUE TITLE
Inserting in place of a selection

### ISSUE THREAD

**Issue by gasparl** [WillemSleegers/tidystats-Google-Docs-add-in#6]
In your current implementation, insertions can only be done if the cursor is placed somewhere in the document and nothing is selected. This is fine by me, but, FYI, I wrote [a custom function](https://github.com/gasparl/tidystats-Google-Docs-add-in/blob/master/src/server/functions/insertPlain.ts) that should work with selections too. I can't guarantee it's 100% bug-free, but still the option is there.

**Comment by WillemSleegers** [comment]
Yeah seems like a good option. Can you get rid of the commented out code and add a little bit more commenting about how it works? Like what the elements are and why they're there and why you need to loop over them.

**Comment by WillemSleegers** [comment]
And then make a PR. Thanks!
